src/page/PlaylistView.jsx

```
@@ -350,50 +411,65 @@ function PlaylistView() {
-                onDrop={(e) => isOwner && handleDrop(e, index)}
-                onDragEnd={() => {
-                  setDraggedIndex(null);
-                  setDragOverIndex(null);
-                }}
+                draggable
+                onDragStart={(e) => handleDragStart(e, index)}
+                onDragOver={(e) => handleDragOver(e, index)}
+                onDragLeave={() => setDragOverIndex(null)}
+                onDrop={(e) => handleDrop(e, index)}
+                onDragEnd={() => { setDraggedIndex(null); setDragOverIndex(null); }}
               >
```

Biome (2.4.6)

[error] Static Elements should not be interactive.

(lint/a11y/noStaticElementInteractions)